src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -153,6 +156,33 @@ class DocBaseViewer extends BaseViewer {
         super.destroy();
     }
 
+    /**
+     * Converts a value and unit to page number
+     *
+     * @return {number|undefined} a page number > 0
+     */
+    getStartPage() {
+        let convertedValue;
+
+        const { unit, value } = this.fileStartObj || {};
```

Comment:
If you do `this.startAt = getProp(options,fileOptions.${options.file.id}, {});` above, you don't need to default again to {} here.

Reply:
True, was just being doubly sure